File: Results/DER/Translation/translation_output/w_test/codeqwen/avatar/python/java/atcoder_ABC164_A.java
Here is the Java version of your Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int S = scanner.nextInt();
        int W = scanner.nextInt();
        if (S <= W) {
            System.out.println("unsafe");
        } else {
            System.out.println("safe");
        }
    }
}
```

This Java code does the same thing as your Python code. It reads two integers from the user, compares them, and prints "unsafe" if the first integer is less than or equal to the second, and "safe" otherwise.

File: Results/DER/Translation/translation_output/w_test/codeqwen/avatar/java/python/atcoder_ABC164_A.py
Here is the Python equivalent of the Java code:

```python
def main():
    Sheep, Wolve = map(int, input().split())

    if Wolve >= Sheep:
        print("unsafe")
    else:
        print("safe")

if __name__ == "__main__":
    main()
```

This Python code does the same thing as the Java code. It reads two integers from the user, then checks if the second integer (Wolve) is greater than or equal to the first integer (Sheep). If it is, it prints "unsafe". Otherwise, it prints "safe".
